### ISSUE TITLE
The toolbar in image preview mode lacks hovertips

### ISSUE THREAD

**Issue by joebeijing** [toeverything/AFFiNE#2788]
![CleanShot 2023-06-15 at 13 19 04@2x](https://github.com/toeverything/AFFiNE/assets/22443345/d9e47d0b-2738-4f78-a56d-f2b8b2bd0965)
- Previous
- Next
- Fit to screen
- Zoom out
- Zoom in
- Download
- Copy to clipboard
- Delete

Also, a toast "Copied to clipboard" should pop up after clicking the "Copy to clipboard" button.
```[tasklist]
### Tasks
- [x] Tooltip implementation
- [x] Toast for copied to clipboard
```

**Comment by danielchim** [comment]
Will be available in the next PR.

**Comment by danielchim** [comment]
![chrome_MQMFQiW2lw](https://github.com/toeverything/AFFiNE/assets/12156547/6d7b90f4-8902-4eb3-adae-7fbfa7c6535d)

**Comment by danielchim** [comment]
Available in [0.7.0-canary.19](https://github.com/toeverything/AFFiNE/releases/tag/v0.7.0-canary.19)
